src/Xamarin.Android.Build.Tasks/Tasks/BundleToolBuildBundle.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Xamarin.Android.Tasks
+{
+	/// <summary>
+	/// Invokes `bundletool` to create an Android App Bundle (.aab file)
+	/// 
+	/// Usage: bundletool build-bundle --modules=base.zip --output=foo.aab --config=BundleConfig.json
+	/// </summary>
+	public class BundleToolBuildBundle : BundleTool
```

Comment:
Would `<CreateBundle/>` be a more semantically meaningful name?

Reply:
👍  `<BuildAppBundle/>`, then `<BuildApkSet/>` and `<InstallApkSet/>`.